src/main/java/com/finance/app/model/entity/Category.java

```diff
@@ -0,0 +1,22 @@
+package com.finance.app.model.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Collection;
+
+@Entity
+@Data
+@NoArgsConstructor
+@Table(name = "categories")
+public class Category {
```

Comment:
https://thorben-janssen.com/ultimate-guide-to-implementing-equals-and-hashcode-with-hibernate/

https://docs.google.com/document/d/1cshcb0Pe8BZRv9MqbchA2OivICnWNRBaZ2F70Id_VT4/edit